cmd/node/config/enableEpochs.toml

```diff
@@ -236,6 +236,9 @@
     # WipeSingleNFTLiquidityDecreaseEnableEpoch represents the epoch when the system account liquidity is decreased for wipeSingleNFT as well
     WipeSingleNFTLiquidityDecreaseEnableEpoch = 5
 
+    # DynamicGasCostForDataTrieStorageLoadEnableEpoch represents the epoch when dynamic gas cost for data trie storage load will be enabled
+    DynamicGasCostForDataTrieStorageLoadEnableEpoch = 6
```

Comment:
switch the value to 3? (what will come from rc/v1.4.0 will have all the other values set to 2)

Reply:
Done.